app/pages/terms/terms.js

```diff
@@ -13,281 +13,173 @@
-
-  renderForm() {
-    var termsUpdated = this.renderTermsUpdated();
-    var ageConsent = this.renderAgeConsentStep();
-    var agreeConfirmation = this.renderAgreeCheckboxes();
-    var continueBtnDisabled = this.getTermsAndPrivacyButtonState();
-    var continueBtnClass = (this.state.ageSelected === this.props.ages.NOT_OF_AGE.value)
-      ? 'terms-button-hidden' : 'terms-button';
-
-    var termsForm;
-    if (this.props.authenticated && !this.props.acceptedLatestTerms) {
-      termsForm = (
-        <form className='terms-form'>
-          {termsUpdated}
-          {ageConsent}
-          {agreeConfirmation}
-          <button
-            className={continueBtnClass}
-            onClick={this.handleTermsAndPrivacySubmit.bind(this)}
-            disabled={continueBtnDisabled}>Continue</button>
-        </form>
-      );
-    }
+  /**
+   *
+   * @param {boolean} agreed True when the checkbox is checked.
+   */
+  renderCheckboxAgreedTerms(agreed) {
+    const branding = CONFIG[config.BRANDING];
+    const {
+      termsText: textTermsOfUse,
+      privacyText: textPrivacyPolicy,
+      terms: urlTermsOfUse,
+      privacy: urlPrivacyPolicy
+    } = branding;
+
+    // TODO Reuse <Trans /> from react-i18next after update to v11.6.0+
+    // https://react.i18next.com/latest/trans-component#alternative-usage-v-11-6-0
     return (
-      <div>
-        {termsForm}
-      </div>
-    );
-  }
-
-  renderAgreeCheckboxes() {
-    var onBehalf;
-    if (this.state.ageSelected === this.props.ages.WITH_CONSENT.value) {
-      onBehalf = (
-        <label htmlFor='agreedOnBehalf'>
-          <input
-            id='agreedOnBehalf'
-            type='checkbox'
-            className='js-terms-checkbox'
-            checked={this.state.agreedOnBehalf}
-            onChange={this.handleOnBehalfAgreementChange.bind(this)} />
-          {this.props.messages.ACCEPT_ON_BEHALF}
-        </label>
-      );
-    }
-
-    var acceptTerms;
-    if (this.state.ageSelected !== this.props.ages.NOT_OF_AGE.value) {
-      acceptTerms = (
-        <label htmlFor='agreed'>
+      <p id="checkbox-agreed-terms">
+        <label id="label-checkbox-agreed-terms" htmlFor='input-agreed-terms'>
           <input
-            id='agreed'
+            id='input-agreed-terms'
             type='checkbox'
             className='js-terms-checkbox'
-            checked={this.state.agreed}
-            onChange={this.handleAgreementChange.bind(this)} />
-          {this.props.messages.ACCEPT_OF_AGE}
+            checked={agreed}
+            onChange={() => this.setState({ agreed: !this.state.agreed })} />
+          &nbsp;
+          <span>
+            {t('I am 18 or older and I accept the terms of the')}
+            &nbsp;
+            <a href={urlTermsOfUse} target='_blank' rel='noreferrer'>{textTermsOfUse}</a>
+            &nbsp;
+            {t('and the')}
+            &nbsp;
+            <a href={urlPrivacyPolicy} target='_blank' rel='noreferrer'>{textPrivacyPolicy}</a>
```

Comment:
Not sure that kind of patterns can be translated correctly in any language if we don't use the Trans component.